wordpress-mobile/WordPress-Android
# 🤖 Use multiple avatars for multiple new likes or subscribers

Fixes #20039

## Description
This PR adds multiple avatars for multiple post or comment likes and multiple subscribers  

|Light mode|Dark mode|
|---|---|
|![Screenshot_20240207_164315](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/42ce50a9-b334-4fea-9930-05458fea64dd)|![Screenshot_20240207_164327](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/18f0e2de-60ef-4c31-b2e4-cefde84ac061)|

|Landcape|RTL|
|---|---|
|![Screenshot_20240207_164346](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/1bbc3417-278f-4600-bcaf-7da0ec771ab6)|![Screenshot_20240207_164414](https://github.com/wordpress-mobile/WordPress-Android/assets/304044/f9ed566e-5440-414d-9b91-15df387dde73)|

-----

## To Test:

1. Open the notification tab in the Jetpack app
2. *Verify* that 2 avatars appear for 2 likes/subscribes
3. *Verify* that 3 avatars appears for more than 2 likes/subscribes 

-----

## Regression Notes

1. Potential unintended areas of impact

    - Notifications tab

4. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

5. What automated tests I added (or what prevented me from doing so)

    - We plan to add tests after refactoring this part of the code
   
-----

## PR Submission Checklist:


-----

## Testing Checklist:
